src/components/certificates/CertificateForm.tsx

```diff
@@ -156,7 +162,7 @@ const CertificateForm: React.FC<CertificateFormProps> = ({
             </div>
 
             <SelectField
-              label="Certificate type"
+              label="Certificate Type"
```

Comment:
Really small, but this should also be translated. Any text gets translated - descriptive text, labels, headers, placeholders, etc.